### ISSUE TITLE
[Feature] Multiple administrator accounts

### ISSUE THREAD

**Issue by emorchy** [lardbit/nefarious#88]
I am loving this project and all its usefulness, and I wish to recommend a feature regarding administrator accounts. If a user wants to stop watching after they complete a movie or show, one would need to log into an admin account and stop watching from there. Unfortunately, I am unable to grant admin privileges to user accounts.
I believe this would be incredibly useful because multiple users can become administrators, giving the ability to stop watching shows. In addition, if multiple moderators collaborate on this tool without wanting to share the same credentials, multiple administrator accounts would be beneficial.
I wish to propose two possible options towards making this feature possible:
1. A checkbox next to each user to select whether they have privileged access in the nefarious settings page.
2. The less secure option would be to allow users to stop watching their own shows but still prevent access to the settings page. It would not create the feature of having multiple administrators, but it would give extra privileges to multiple user accounts.

**Comment by lardbit** [comment]
This is a good idea regarding the ability to add additional "admin" users.  I will leave this open until I get the chance to add that feature.

However, you also found a bug.  I intended for unprivileged users to be able to stop watching their own shows & movies, but it looks like it wasn't working for tv shows.  I just fixed that and it should be releasing soon.

As a workaround for creating more admin users you can actually log into the "admin console" and add a user there.

Admin console: http://127.0.0.1:8000/admin/
User add: http://127.0.0.1:8000/admin/auth/user/add/

The passwords are more strict when adding from the admin console.  Users added through nefarious will let you set any password.

After you add the user, you'll have to edit the user in the admin console to be *staff status* and *superuser status*.

**Comment by lardbit** [comment]
The ARM builds are currently failing and I think I remember you using that architecture, so it's not releasing my changes yet.

https://travis-ci.com/github/lardbit/nefarious/builds

I'm researching that but wanted to give you a heads up the bug fix isn't releasing for ARM right now.

**Comment by emorchy** [comment]
Thank you so much for your quick response. I was previously unaware of the Django site administration for nefarious, and this helps significantly. I will keep my eye out for any more bugs in the future. Thank you again.

**Comment by lardbit** [comment]
@emorchy, I'm working on building docker images for different architectures.  I know you're running a raspberry pi like me.  Can you tell me which pi you're using and what the following command says?

    uname -m

I'm trying to find out if I need to support 32 or 64 bit architectures.

**Comment by emorchy** [comment]
Absolutely. My raspberry pi is a 3B+. It has a 32-bit architecture with an armv7l processor.
Output: `armv7l`

**Comment by lardbit** [comment]
Thanks.  It surprised me that the pi's are 64 bits but run a 32 bit os.  I'm using travis-ci to automatically build the docker images but they don't support 32-bit arm...so I'm going to start using Github Actions which apparently supports a lot more architectures.

**Comment by emorchy** [comment]
I'm glad I could be of help. Please let me know if there is anything more I could do to further raspberry pi support to this project.
